opendemic/human/model.py

```diff
@@ -383,10 +383,21 @@ def create_human(telegram_human_id: int = None, fingerprint: str = None) -> Tupl
 				mysql_db_format_value(value=fingerprint)
 			)
 		)
-	except Exception as e:
-		logger.error(e)
-		err = e
-	return Human(human_id=human_id), err
+	except Exception as query_creation_err:
+		logger.error(query_creation_err)
+		return None, query_creation_err
+	else:
+		if db_err is not None:
+			logger.error(db_err)
+			return None, db_err
+		else:
```

Comment:
No need for else statement. Just suggesting this because it helps to make the code readable. We either return, or keep going